### ISSUE TITLE
⌨ Additional details description is announced on every line

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#15218]
### ⌨ Accessibility and Expected Behaviour

🛡️Fable tester reported: When I answer the questions about executive opportunities and executive coaching opportunities, the description which contains info about how it is not guaranteed is announced on every line of these questions such as after a choice is read out when focusing on an option in the set of radiobuttons. This makes navigating through the radio buttons overly verbose. 

Expected behaviour: Ideally, it should only be announced as part of the question and not after each choice as you go through the set of radio buttons.

### 📋 Steps to Reproduce

1. Sign in 
2. verify employee email 
3. navigate to career development 
4. complete the section of the form around interest in executive level opportunities, Executive coaching status, and Interest in executive coaching opportunities

### What operating systems are you encountering the problem on?

Apple IOS

### What browsers are you encountering the problem on?

Safari

### What assistive technologies are you encountering the problem on?

VoiceOver

### 📸 Screenshot

<img width="493" height="756" alt="Image" src="https://github.com/user-attachments/assets/ee4a6430-797d-4afd-a2cd-77045e6d7db4" />

### 🙋‍♀️ Proposed Solution

Ideally the special note should only be announced once, after you hear the question.

**Comment by esizer** [comment]
Able to reproduce on iOS + Safari + VoiceOver but not MacOS.

I beleive this is related to some inconsistency in how it is handled. I'm not sure how we fix this, it seems like iOS voiceover is just much more verbose 😢 

The ony solution I know of is to no longer associate the description with the fieldset. However, we would then need to assume people would not have this read out, they would need to manually navigate to it. Is that an acceptable compromise?

**Comment by esizer** [comment]
> I don't think that's a good compromise. I assume users are just tabbing through the form in which case they would never encounter this text.

I agree. While this might be annoying for some, I think that might be better than possibly missing the text entirely. Maybe we can close this as not planned.

Unless we want to give design some time to consider a different solution?
